lib/model/narrow.dart

```diff
@@ -168,6 +168,13 @@ class DmNarrow extends Narrow implements SendableNarrow {
     );
   }
 
+  factory DmNarrow.withUsers(List<int> userIds, {required int selfUserId}) {
```

Comment:
nit in commit message:

    Add DmNarrow.withUsers

Include a subject-area prefix: here `narrow:` would be good.

For discussion, see in the Zulip-wide Git style guide: https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-summary-part-1
The examples there are all for the server and web app, but the principle is the same.